src/index.js

```diff
@@ -393,7 +448,7 @@ var recycleNode = function(node) {
     : createVNode(
         node.nodeName.toLowerCase(),
         EMPTY_OBJ,
-        map.call(node.childNodes, recycleNode),
+        EMPTY_ARR.map.call(node.childNodes, recycleNode),
```

Comment:
@zaceno Why?

Reply:
There was already a function called `map` in `src/index.js` which was just a proxy to `EMPTY_ARR.map`. That `map` was only used once. So when I was going to define my own function named `map`, I got rid of the old definition and replaced the usage with `EMPTY_ARR.map` directly.